plugin/src/main/java/io/snyk/eclipse/plugin/views/snyktoolview/BrowserHandler.java

```diff
@@ -66,6 +67,18 @@ public Object function(Object[] arguments) {
 			}
 		};
 
+		new BrowserFunction(browser, "initiateLogin") {
+			@Override
+			public Object function(Object[] arguments) {
+				SnykWizard wizard = new SnykWizard();
+				WizardDialog dialog = new WizardDialog(Display.getDefault().getActiveShell(),wizard);
+
+				dialog.setBlockOnOpen(true);
+				dialog.open();
+				return null;
```

Comment:
suggestion: don't duplicate the code, but move `SnykExtendedLanguageClient.runSnykWizard()` to a central place and use it from all places where the Wizard is launched.